hudi-client/src/test/java/org/apache/hudi/testutils/HoodieTestDataGenerator.java

```diff
@@ -70,7 +70,9 @@
 public class HoodieTestDataGenerator {
 
   // based on examination of sample file, the schema produces the following per record size
-  public static final int SIZE_PER_RECORD = 50 * 1024;
+  public static final int BYTES_PER_RECORD = (int) (1.2 * 1024);
```

Comment:
Why did the size shrink that much? Was this wrong?

Reply:
yes, this was wrong before.